Sentinel/Classes/Core/Trigger.swift

```diff
@@ -18,16 +21,19 @@ public protocol Trigger {
 
 /// Provides different trigger types based on the event which makes the trigger.
 public enum Triggers {
-    /// The trigger type which is triggered on the shake event.
-    public static var shake: Trigger { ShakeTrigger() }
 
-    /// The trigger type which is triggered on the screenshot event.
-    public static var screenshot: Trigger { ScreenshotTrigger() }
 
     /// The trigger type which is triggered on the specified notification name.
     public static func notification(forName name: Notification.Name) -> Trigger {
         NotificationTrigger(notificationName: name)
     }
+    #if !os(macOS)
```

Comment:
Would it be better to use `#if os(iOS)`? Because with this, e.g. `tvOS` or `watchOS` in the future would support this. I know we will not add the possibility for that, but it is easier to work with constrained things than the opposite.

Reply:
Also, add a note to the comment (or discussion), on which platforms are those methods available.